test/support/stacktraces.js

```diff
@@ -8,6 +8,6 @@ if (!process.env.DEBUG && !process.env.CI) {
 // A hack to deblock them is push some events in the event loop.
 // See https://github.com/electron/electron/issues/7083#issuecomment-262038387
 // And https://github.com/electron/electron/issues/1833
-if (process.platform === 'linux') {
+if (process.platform === 'linux' && !process.env.NO_ELECTRON) {
```

Comment:
:ok_hand: We should probably move this hack somewhere else since it has nothing to do with stacktraces...